docs/_utils/redirections.yaml

```diff
@@ -0,0 +1,258 @@
+### a dictionary of redirections
```

Comment:
This file is not relevant for this project

Reply:
ok I'll remove it